modules/reference/pages/cluster-properties.adoc

```diff
@@ -1,1796 +1,5372 @@
+
+Raft I/O timeout.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `10000ms`
+
+---
+
+=== raft_learner_recovery_rate
+
+Raft learner recovery rate limit. Throttles the rate of data communicated to nodes (learners) that need to catch up to leaders.
+
+This rate limit is placed on a node sending data to a recovering node. Each sending node will be limited to this rate. The recovering node will accept data as fast as possible according to the combined limits of all healthy nodes in the cluster. For example, if two nodes are sending data to the recovering node, and `raft_learner_recovery_rate` is 100 MB/sec, then the recovering node will recover at a rate of 200 MB/sec.
+
+*Units*: bytes per second.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `104857600`
+
+---
+
+=== raft_max_concurrent_append_requests_per_follower
+
+Maximum number of concurrent append entries requests sent by leader to one follower.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`0`, `4294967295`]
+
+*Default:* `16`
+
+---
+
+=== raft_max_recovery_memory
+
+Max memory that can be used for reads in raft recovery process by default 15% of total memory.
+
+*Requires Restart:* No
+
+*Nullable:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `null`
+
+---
+
+=== raft_recovery_concurrency_per_shard
+
+How many partitions may simultaneously recover data to a particular shard. This is limited to avoid overwhelming nodes when they come back online after an outage.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `64`
+
+---
+
+=== raft_recovery_default_read_size
+
+Default size of read issued during raft follower recovery.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `524288`
+
+---
+
+=== raft_recovery_throttle_disable_dynamic_mode
+
+Disables dynamic rate allocation in recovery throttle (advanced).
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== raft_replica_max_pending_flush_bytes
+
+Max not flushed bytes per partition. If configured threshold is reached log will automatically be flushed even though it wasn't explicitly requested.
+
+*Requires Restart:* No
+
+*Nullable:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `262144`
+
+---
+
+=== raft_replicate_batch_window_size
+
+Max size of requests cached for replication.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `1048576`
+
+---
+
+=== raft_smp_max_non_local_requests
+
+Maximum number of x-core requests pending in Raft seastar::smp group. (for more details look at `seastar::smp_service_group` documentation).
```

Comment:
**ENG:** Are these supposed to be links to Seastar docs? Only topic related to this that I could find is https://docs.seastar.io/master/classseastar_1_1smp__service__group.html. Please clarify the destination for these links.